TESTING.md

```diff
@@ -0,0 +1,26 @@
+# TESTING mypy
+The basic way to run all tests:
+
+    $ pip3 install -r test-requirements.txt
+    $ python2 -m pip install -U typing
+    $ ./runtests.py
+
+Mypy is tested using `pytest`. This can be installed as shown above, using `test-requirements.txt`. In essence, there are multiple kinds of tests for mypy:
```

Comment:
Again, please break all lines longer than 72 characters (that's a soft limit, but they can't be longer than 79 characters unless there's a single very long URL).